packages/web/src/components/PopupModal/PopupModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, PropsWithChildren } from 'react';
+import {
+  Button,
+  ButtonProps,
+  HStack,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { popupProps } from './utils';
+
+interface PopupModalProps extends PropsWithChildren {
+  openModalText: string;
+  header: string;
+  hideCloseButton?: boolean;
+  openButtonProps?: ButtonProps;
```

Comment:
This value is being passed around all over the place and I don't think we should even be supporting this; let's scrap both the original and the alias's usage. If you want the button in the modal to pop, use `variant="cta"`.